dockerfiles/jenkins-runner/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jenkins/jenkinsfile-runner:adoptopenjdk-11-hotspot
+
+# Install git, docker, sdkman
+RUN apt-get update && \
+  apt-get install -y wget git unzip zip apt-transport-https ca-certificates curl gnupg2 software-properties-common && \
+  curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg > /tmp/dkey; apt-key add /tmp/dkey && \
+  add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+    $(lsb_release -cs) \
+    stable" && \
+  apt-get update && \
+  apt-get -y install docker-ce &&  \
```

Comment:
Q: 
Why do we need docker (docker in docker) here?
I think we need only the sbt?

Reply:
the gatling integration tests need to have a tmail memory docker container running to be able to execute :)

Reply:
understood, thanks